### PR TITLE
feat: Started creating a Contact Us page for the VWS

### PR DESCRIPTION
**Pull request by lkinch**
For a timely review/response, please avoid force-pushing additional
  commits if your PR already received reviews or comments.
  
  Before submitting a Pull Request, please ensure you've done the following:
  - 📖 Read the VWS Contributing Guide: https://github.com/CodeItQuick/VWS-Open-Source/blob/HEAD/CONTRIBUTING.md.
  - 📖 Read the VWS Main README File: https://github.com/CodeItQuick/VWS-Open-Source/blob/HEAD/README.md.
  - 👷‍♀️ Create small PRs. In most cases, this will be possible.
  - ✅ Provide tests for your changes.
  - 📝 Use descriptive commit messages.
  - 📗 Update any related documentation and include any relevant screenshots.

## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [X] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This is the skeleton frame for the Contact us page. It is very barren, and only using temporary information, but it will look better once CSS gets integrated.

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: src/resources/views/welcome.blade.php [hunk 1]
```diff
@@ -2,76 +2,39 @@
...
-	<style >
-
-	</style>
+<h1>Contact Us</h1>
+<p></p>
+<p>At the Clinic of Dr. Montelpare, we wish to help everyone who seeks help.</p>
+<p></p>
+<p>As such, we would like to extend an invitation for anyone who wishes to meet with us either at our address:</p>
+<p></p>
+<b>11 Mafeking Parade<br>
```
Comment (CodeItQuick) [inline src/resources/views/welcome.blade.php:16]: Can you update to the UPEI address?

File: nginx/default.conf [hunk 2]
```diff
@@ -5,7 +5,7 @@ server {
 	index index.php index.html;
     error_log  /var/log/nginx/error.log;
     access_log /var/log/nginx/access.log;
-	root /var/www/public/;
+	root /var/www/views/;
```
Comment (CodeItQuick) [inline nginx/default.conf:8]: public works for me, and its served out of src/public. Can you merge release-v1.0 back into your branch locally, and see if this is still needed?

**Comment by CodeItQuick** [comment]
blank

**Review by CodeItQuick — CHANGES_REQUESTED** [review]
todo:
1. update to the UPEI address
2. merge release-v1.0 locally into the branch
3. create a contactus.blade.php file and route /contact-us to the page. I can do that if you want.

File: nginx/default.conf (hunk 2 above)
Comment (lkinch) [reply nginx/default.conf:8]: Resolved.

File: src/resources/views/welcome.blade.php (hunk 1 above)
Comment (lkinch) [reply src/resources/views/welcome.blade.php:16]: Resolved.

**Review by lkinch — COMMENTED** [review]
The www/views will be replaced with www/public when the next PR arrives.

File: src/resources/views/welcome.blade.php (hunk 1 above)
Comment (CodeItQuick) [reply src/resources/views/welcome.blade.php:16]: I think you need to git add . then git push here, its not updated.

**Comment by CodeItQuick** [comment]
This PR has been abandoned. Liam updated the repository later anyhow with these changes, he just didn't push on the original PR.

**Pull request closed without merging**